morpheus/cli.py

```diff
@@ -125,8 +124,6 @@ def new_func(*args, **kwargs):
 
             kwargs = _without_empty_args(kwargs)
 
-            CppConfig.set_should_use_cpp(kwargs.pop("use_cpp", CppConfig.get_should_use_cpp()))
```

Comment:
This seems unrelated. Is this being removed intentionally, and why?

Reply:
This was a merge error which I tried to fix this this PR at the same time but I forgot to add it back into the `run` command (Good catch because I had the fix wrong).

This value should only be set in the `run` function which actually has the `--use_cpp` option. Setting it in `prepare_command` will just set the same value over and over again.